build/update-apps.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
https://stackoverflow.com/questions/16365130/what-is-the-difference-between-usr-bin-env-bash-and-usr-bin-bash

Reply:
@kesselb So according to that post, there are benefits but also drawbacks (e.g. if we ever want to pass arguments like branches etc). Does this change actually fix an issue, like not working on a specific OS?

Reply:
> Does this change actually fix an issue, like not working on a specific OS?

No ;) Go ahead with `#!/bin/bash`. If someone complains we can change it later.

Reply:
Hehe :) Ok, will resolve this and the other comment then. Does the rest look good? If so feel free to add your approval review.